src/main/java/com/spotinst/sdkjava/model/api/ocean/kubernetes/ApiClusterStrategyConfiguration.java

```diff
@@ -19,6 +19,7 @@
     private Boolean                    utilizeReservedInstances;
     private Integer                    drainingTimeout;
     private Integer                    gracePeriod;
+    private Integer                    spotPercentage;
```

Comment:
Any particular reason for adding this field in the existing cluster level class. If the parameters are different, we should create another class for VNG level.